renderlib/CCamera.cpp

```diff
@@ -2,6 +2,81 @@
 
 #include "gesture/gesture.h"
 
+void
+CCamera::SetViewMode(const EViewMode ViewMode)
+{
+  if (ViewMode == ViewModeUser)
+    return;
+
+  glm::vec3 ctr = m_SceneBoundingBox.GetCenter();
+  m_Target = ctr;
+  m_Up = glm::vec3(0.0f, 1.0f, 0.0f);
+
+  const float size = m_SceneBoundingBox.GetDiagonalLength();
+  const float Length = (m_Projection == ORTHOGRAPHIC) ? 2.0f : size * 0.5f / tan(0.5f * m_FovV * DEG_TO_RAD);
+  m_OrthoScale = DEF_ORTHO_SCALE;
+
+  // const float Distance = 0.866f;
+  // const float Length = Distance * m_SceneBoundingBox.GetMaxLength();
```

Comment:
I know this was just copied from CCamera.h, but why are there these two lines commented out here?

Reply:
I can get rid of them - it's just a remnant of a previous way of trying to calculate the camera distance for these views.